demos/notebooks/dqx_quick_start_demo_library.ipynb

```diff
@@ -0,0 +1,210 @@
+    "    Row(id=6,  age=123, country='France') # Invalid -> age - GT 120\n",
+    "]\n",
+    "\n",
+    "new_users_df = spark.createDataFrame(new_users_sample_data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Demonstrating Functions\n",
+    "- `is_not_null_and_not_empty`\n",
+    "- `is_in_range`\n",
+    "- `is_in_list`\n",
+    "\n",
+    "You can find a list of all available built-in checks in the documentation [here](https://databrickslabs.github.io/dqx/docs/reference/quality_rules/) and in the code [here](https://github.com/databrickslabs/dqx/blob/main/src/databricks/labs/dqx/check_funcs.py).\n",
+    "\n",
+    "We are demonstrating creating and validating a set of `Quality Checks` defined declaratively using YAML.\n",
+    "\n",
+    "We can use `validate_checks` to verify that the checks are defined correctly."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "checks_from_yaml = yaml.safe_load(\"\"\"\n",
+    "- check:\n",
+    "    function: is_not_null_and_not_empty\n",
+    "    for_each_column:  # define check for multiple columns at once\n",
+    "      - id\n",
+    "      - age\n",
+    "      - country\n",
+    "  criticality: error\n",
+    "- check:\n",
+    "    function: is_in_range\n",
+    "    arguments:\n",
+    "      column: age  # define check for a single column\n",
+    "      min_limit: 18\n",
+    "      max_limit: 120\n",
+    "  criticality: warn\n",
+    "  name: age_not_in_range  # optional check name, auto-generated if not provided\n",
+    "- check:\n",
+    "    function: is_in_list\n",
+    "    for_each_column:\n",
+    "      - country\n",
+    "    arguments:\n",
+    "      allowed:\n",
+    "        - Germany\n",
+    "        - France\n",
+    "  criticality: warn\n",
+    "\"\"\")\n",
+    "\n",
+    "# Validate YAML checks\n",
+    "status = DQEngine.validate_checks(checks_from_yaml)\n",
+    "print(f\"Checks from YAML: {status}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "vscode": {
+     "languageId": "polyglot-notebook"
+    }
+   },
+   "source": [
+    "### Setup `DQEngine`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ws = WorkspaceClient()  # auto-authenticated inside Databricks\n",
+    "dq_engine = DQEngine(ws)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Apply Rules\n",
+    "`apply_checks_by_metadata` results in one `DataFrame` with `_errors` and `_warnings` metadata columns added."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "validated_df = dq_engine.apply_checks_by_metadata(new_users_df, checks_from_yaml)\n",
+    "validated_df.show(truncate=False)"
```

Comment:
is it possible to use display(validated_df) in visual studio code or only works when the notebook is run on Databricks?